schemas/common/relationshipdescriptor.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/relationshipdescriptor",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Relationship Descriptor",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/schemadescriptor#/definitions/descriptor"
+  ],
```

Comment:
I'm not sure we need an inheritance graph across descriptors. They have almost nothing in common. I would prefer to copy `source...` part there, and not to use `source` prefix in the identity descriptor.

Reply:
A descriptor at minimum needs to say what schema it is a descriptor for.  Why would we have a different way to do this for each descriptor type?

Reply:
But the relationship descriptor belongs to two schemas, expecially in many-to-many case. But that's not that important.